VideoApp/VideoApp/NewCode/Screens/Room/RoomViewModel.swift

```diff
@@ -16,7 +16,7 @@
 
 import Combine
 
-class RoomViewModel: ObservableObject {
+@MainActor class RoomViewModel: ObservableObject {
```

Comment:
Had to use this on a few classes because this line was getting called on a background thread:

https://github.com/twilio/twilio-video-app-ios/blob/3e0ed027416d9ea1c75936fb3da7342d4a03d42a/VideoApp/VideoApp/NewCode/Screens/Room/RoomViewModel.swift#L73

It wasn't really an issue until I made changes in this PR to make `RoomManager` publish more state. Then Xcode gave me a nice thread warning.

`@MainActor` is pretty cool. I basically makes sure all properties and methods are called on the main thread.